tests/integration/test_vulnerability_detector/test_scan_results/data/test_cases/cases_no_agent_data.yaml

```diff
@@ -33,6 +33,14 @@
   metadata:
     system: ALAS2
 
+
+- name: ALAS 2023
+  description: Scan ALAS vulnerabilities using only the NVD feed
+  configuration_parameters:
+    NVD_JSON_PATH: CUSTOM_NVD_JSON_PATH
+  metadata:
+    system: ALAS_2023
+
```

Comment:
For this case, let's leave only the ALAS2 case, or else overwrite it by ALAS2023.

Reply:
Done in commit: refactor(#4440): Reduce the number of ALAS cases to one in some tests